test-prow-e2e.sh

```diff
@@ -4,6 +4,7 @@ set -exuo pipefail
 
 ARTIFACT_DIR=${ARTIFACT_DIR:=/tmp/artifacts}
 SCREENSHOTS_DIR=integration-tests/screenshots
+INSTALLER_DIR=${INSTALLER_DIR:=${ARTIFACT_DIR}/installer}
```

Comment:
Oops! Mistakenly removed the `INSTALLER_DIR` recently, actually it is used here - https://github.com/openshift/console-plugin-template/blob/main/test-prow-e2e.sh#L20